packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -103,6 +107,7 @@ const RekognitionAndTextract = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': 'us-east-1',
   'ap-northeast-2': latestTextractRegions >= 1 ? 'ap-northeast-2' : 'us-east-1',
+  'ap-northeast-3': 'ap-northeast-2',
```

Comment:
Textract is not available in ap-northeast-3. Falling back to ap-northeast-2.

https://docs.aws.amazon.com/general/latest/gr/textract.html